config.py

```diff
@@ -22,10 +22,31 @@ def next_year():
         config = Configuration(year=new_year)
         db.session.add(config)
         db.session.commit()
-        Configuration.update_current_year(config.id)
 
     except Exception as e:
         db.session.rollback()
         raise e
 
     return redirect(url_for("index"))
+
+# This method allows the admin to move on to the next academic year for all users.
+@config_bp.route('/next_year', methods=['POST'])
+@login_required
+@check_access_level(Role.ADMIN)
+def next_year():
```

Comment:
@luciledierckx Using this implementation we'll always be going in one way. Is there any case for which we'd like to revert and go to the past ?

Reply:
The only reason we would like to revert would be that we mistakenly call the next_year functionality more than once a year. As this action is available for all admins, maybe it would be safer to have an option to go back in case one of them mistakenly calls it?